.github/scripts/label_promoted_commits.py

```diff
@@ -77,9 +77,12 @@ def main():
                     comment = f'Closed via {commit.sha}'
                     add_comment_and_close_pr(pr, comment)
             else:
-                print(f'master branch, pr number is: {pr_number}')
-                pr = repo.get_pull(pr_number)
-                pr.add_to_labels('promoted-to-master')
+                try:
+                    pr = repo.get_pull(pr_number)
+                    pr.add_to_labels('promoted-to-master')
+                    print(f'master branch, pr number is: {pr_number}')
+                except:
+                    pass
```

Comment:
We'll never know something bad happened. Print the exception at least somewhere?

Reply:
the `pass` statement is redundant.

Reply:
removed